package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Crash or abort if you get an unhandledRejection or multipleResolves",
   "main": "make-promises-safe.js",
   "scripts": {
-    "test": "standard | snazzy && tap test.js"
+    "test": "standard --fix | snazzy && tap test.js"
```

Comment:
Can you revert this? If you want, please put it in a separate script. We run `test` before any commit and CI.

Reply:
reverted